test/footer.test.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {shallow, mount} from 'enzyme';
+import Footer from '../src/client/Footer';
+
+describe('<Footer />', () => {
+  jest.useFakeTimers();
+  const wrapper = shallow(<Footer />);
+
+  describe('When users connect to the website', () => {
+    it('Footer should display', () => {
```

Comment:
We'll always make the `description` be a sentence. This is kind of pattern or coding style.
If we say `it Footer should display`, it might not be such that readable for everyone.

For example, `it should display Footer` or `it should tick the clock`.
It will be much more like a casual sentence and more readable.

Also, that's why the function is named as `it`.